onnxruntime/core/mlas/lib/sqnbitgemm_kernel_avx2.cpp

```diff
@@ -434,6 +434,44 @@ SQ4BitGemmM1Kernel_CompInt8_avx2(
     }
 }
 
+size_t
+SQ4BitGemmKernel_CompInt8_avx2(
```

Comment:
This make sense - the api becomes simple. avx can choose what to do internally without breaking neon.